src/borg/locking.py

```diff
@@ -21,6 +23,29 @@ def get_id():
     return _hostname, _pid, thread_id
 
 
+def check_lock_data(host, pid, thread):
+    """Check if the host, pid, thread combination corresponds to a dead process on our local node or not."""
+    if host != _hostname:
+        return False
+
+    try:
+        # This may not work in Windows.
```

Comment:
Since windows port is in the works, I feel like potentially questionable Windows-related bits should be at least highlighted to be checked as the port is progressing.